Sourcery/Models/Type.swift

```diff
@@ -24,23 +24,25 @@ class Type: NSObject {
     /// Name in parent scope
     var localName: String
 
-    /// All type static variables
-    var staticVariables: [Variable]
-
     /// All instance variables
     var variables: [Variable]
 
     /// Annotations, that were created with // sourcery: annotation1, other = "annotation value", alterantive = 2
     var annotations: [String: NSObject] = [:]
 
+    /// All type static variables
+    var staticVariables: [Variable] {
+        return variables.filter({ $0.isStatic })
+    }
+
     /// Only computed instance variables
     var computedVariables: [Variable] {
-        return variables.filter { $0.isComputed }
+        return variables.filter { $0.isComputed && !$0.isStatic }
```

Comment:
We need to, my first version did that but that's a simplification that's not going to work in more advanced cases eg. you can have let that doesn't have write, you have bodies that don't necessarily mean computed like didSet etc